lib/ui/about/license_list_page.dart

```diff
@@ -0,0 +1,63 @@
+import 'dart:convert';
+
+import 'package:flutter/services.dart';
+import 'package:reservation_system_customer/ui/about/license_detail_page.dart';
+import '../../ui_imports.dart';
+import 'license.dart';
+
+class LicenseListPage extends StatefulWidget {
+  @override
+  _LicenseListPageState createState() => _LicenseListPageState();
+}
+
+class _LicenseListPageState extends State<LicenseListPage> {
+  List licenses = [];
+
+  _getLicenses() async {
+    List tmp = [];
+    Map<String, dynamic> jsonLicenses =
+        json.decode(await rootBundle.loadString("assets/licenses.json"));
+
+    if (jsonLicenses != null) {
+      jsonLicenses.forEach((key, value) {
+        tmp.add(License(
+          libName: key,
+          licenseText: value,
+        ));
+      });
+
+      setState(() {
+        licenses = tmp;
+      });
+    }
+  }
```

Comment:
You could also switch this to a map:
```suggestion
_getLicenses() async {
    Map<String, dynamic> jsonLicenses =
        json.decode(await rootBundle.loadString("assets/licenses.json"));

    if (jsonLicenses != null) {
      final loadedLicenses = jsonLicenses.entries
          .map((entry) => License(
                libName: entry.key,
                licenseText: entry.value,
              ))
          .toList();

      setState(() {
        licenses = loadedLicenses;
      });
    }
  }
```